SS2-Project/Assets/Starstorm2/Modules/Characters/Survivors/Knight.cs

```diff
@@ -88,6 +116,70 @@ public void ModifyStatArguments(RecalculateStatsAPI.StatHookEventArgs args)
                 args.moveSpeedReductionMultAdd += 0.6f;
             }
         }
+
+
+        public class KnightSpecialEmpowerBuff : BaseBuffBehaviour, IBodyStatArgModifier
```

Comment:
Replace class with a single hook on RecalculateSTatsAPI.GetstatCoefficients. This way we replace the monobehaviour with just a method

Reply:
let me add a TODO to do this since I rather not be changing implementations until we actually get this all ported and testing